package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "resin-multibuild",
+  "version": "0.0.1",
+  "description": "Build a docker composition",
+  "main": "build/index.js",
+  "types": "lib/index.ts",
+  "scripts": {
+    "prepublish": "npm run build && npm run test",
+    "precommit": "npm run lint && npm run test",
+    "lint": "tslint '{src,typings}/**/*.ts'",
+    "build": "tsc",
+    "docgen": "typedoc --ignoreCompilerErrors --out docs --name resin-multibuild --readme README.md lib/",
+    "test": "npm run lint && ts-mocha test/*.ts --timeout 60000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/resin-io-modules/resin-multibuild.git"
+  },
+  "author": "Cameron Diver <cameron@resin.io>",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/resin-io-modules/resin-multibuild/issues"
+  },
+  "homepage": "https://github.com/resin-io-modules/resin-multibuild#readme",
+  "devDependencies": {
+    "@types/mocha": "^2.2.44",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "gulp": "^3.9.1",
```

Comment:
It seems you can remove gulp, related deps and gulpfile.

Reply:
Oh yeah, I was that happy to get circle behaving that I forgot about this haha.